urllib3/response.py

```diff
@@ -150,7 +150,7 @@ def get_redirect_location(self):
         return False
 
     def release_conn(self):
-        if not self._pool or not self._connection:
+        if not self._pool or self._connection is None:
```

Comment:
Is there any benefit to changing `not self._connection` to `self._connection is None`? I feel like the former is safer, no?